README.md

```diff
@@ -18,18 +23,47 @@ As a quantum circuit compiler, it is fully aware of the semantics of each gate a
 It understands the quantum programming language cQASM 3 and will support additional quantum programming languages in the future.
 It is developed in modern Python and follows best practices.
 
+## User manual
+
+OpenSquirrel documentation is hosted through GitHub Pages [here](https://QuTech-Delft.github.io/OpenSquirrel/).
```

Comment:
```suggestion
[User documentation of OpenSquirrel](https://QuTech-Delft.github.io/OpenSquirrel/) is hosted through GitHub Pages.
```

Reply:
I'm leaving it like: `User documentation of OpenSquirrel is hosted through [GitHub Pages](https://QuTech-Delft.github.io/OpenSquirrel/).`

Reply:
Wouldn't be more clear to the user to link from 'User documentation of OpenSquirrel' instead of linking from 'GitHub pages', the latter is the means by which we publish, and the former is actually what the link points to.

Reply:
Done!